sites/docs/src/content/docs/checklists/community_governance/maintainers_team.mdx

```diff
@@ -34,3 +34,16 @@ To review mainterins team membership:
 4. [ ] Once accepted, nominee and team leads perform tasks [onboarding](#maintainers-team-onboarding).
 5. [ ] Move members to alumni status on the website in [**About > Governance > Maintainers**](https://nf-co.re/governance#maintainers).
 6. [ ] Announce new members on [#announcements](https://nfcore.slack.com/archives/CE6P95170).
+
+## Helpdesk tasks
+
+When hosting a running helpdesk (doing this is not exclusive to maintainters)! We suggest the following steps:
+
+- Monthly basis
+    - [ ] Check to see for open slots, and make sure sign up on weekly signup sheet (see pin on weekly organisation channel)
+- When hosting (on the day)
+    - [ ] 1h before, advertise on social media (core) and on #weekly-helpdesk
+    - [ ] Sit on gather or huddle for one hour
+    - [ ] Afterwards dump detailed notes in the hackamd
+    - [ ] Post a short summary on #weekly-helpdesk
+    - [ ] Send a reminder to next weeks hosts they are up
```

Comment:
Though, I don't know whether this would be very findable as in under the maintainers section
```suggestion
    - [ ] Send a reminder to next weeks hosts they are up
    
:::note
Hosting a helpdesk is not exclusive to maintainers and we encourage community members interested to get involved
:::
```